### PR TITLE
docs: updated Getting Started Guide

### PR DESCRIPTION
**Pull request by Adeniyikayodee**
# Reason for This Change

Elaborate on the reasons for this change. If applicable link it
to the related issue (like `Closes #<issue>`).

Note: We are not accepting feature PR's for the time being (this will
change in the future).

## Description of Changes

Overview of what was changed.

### CODE REVIEW

**Comment by soerenmartius** [comment]
@Adeniyikayodee please remove the changes in `docs/pnpm-lock.yaml` from this Pull Request! Thank You!

File: docs/getting-started/index.md [hunk 1]
```diff
@@ -15,18 +15,21 @@ next:
 
 [Terramate](https://github.com/mineiros-io/terramate) is a [code generator and orchestrator for Terraform](https://blog.terramate.io/product-introduction-github-as-code-af466550a4a9?source=friends_link&sk=ae60be77dcb484724b3b821898e7813d) that adds powerful capabilities such as code generation, stacks, orchestration, change detection, data sharing and more to Terraform.
 
-This tutorial will demonstrate the basic concepts behind [Terramate](https://github.com/mineiros-io/terramate) and give you some ideas for how a filesystem-oriented code generator can help you to manage your Terraform code at scale more efficiently .
-So that anyone can try Terramate without needing a cloud account, we will use only the `local_file` resource to create a static site demonstrating the basic principles behind Terramate. The only prerequisites are local installations of Terramate, Terraform and Git.
+This tutorial aims to introduce you to the basic concepts of [Terramate](https://github.com/mineiros-io/terramate), and demonstrate how a filesystem-oriented code generator can improve the management of your Terraform code efficiently.
```
Comment (i4ki) [inline docs/getting-started/index.md:18]:
```suggestion
This tutorial aims to introduce you to the basic concepts of [Terramate](https://github.com/terramate-io/terramate), and demonstrate how a filesystem-oriented code generator can improve the management of your Terraform code efficiently.
```

**Review by soerenmartius — CHANGES_REQUESTED** [review]
Please resolve the conflicts

**Review by soerenmartius — APPROVED** [review]
lgtm!

**Pull request merged**